loadbalancer/fadein_test.go

```diff
@@ -146,33 +146,13 @@ func testFadeIn(
 			}
 
 			// Print CSV-like output for, where row number represents time and
-			// column represents endpoint.
+			// column represents endpoint. You can vizualize it using
+			// ./skptesting/run_fadein_test.sh from the skipper repo root.
 			t.Log("CSV " + fmt.Sprintf("%d,", i) + strings.Join(showStats, ","))
 		}
 	})
 }
 
-// For each endpoint, return a hash key which will make the consistent hash algorithm select it.
```

Comment:
Why is this test not valid anymore?

Reply:
After random keys are used here
https://github.com/zalando/skipper/pull/2363#discussion_r1223200218
this function is not called nowhere.